news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: We are working on a new front-office theme
+subtitle: Here are some informations about it!
+date: 2022-01-13
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.0]
+---
+
+# The PrestaShop team is working on a new theme
+
+For a lot of time, the community asked us to rework the classic theme because a lot of things are missing inside it. We announced that we are working on a new front-office theme that you can find [here](https://github.com/PrestaShop/theme-refacto/compare/develop...mparvazi:auto-rtl).
+
+## Why such a decision?
+
+We received a lot of feedback from the community:
+
+-   Upgrade to Bootstrap 5
+-   Rework the structure inside  `src`  directory, which was  `_dev`  before.
```

Comment:
```suggestion
-   Rework the theme's structure, put `_dev` inside `src` directory.
```

Reply:
switching from _dev to src is not a suggestion from the community, but the structure inside it was